demo-site/templates/origin/origin_app.js

```diff
@@ -0,0 +1,73 @@
+const path = require('path');
+const { env } = require('process');
+const express = require('express');
+const cookieParser = require('cookie-parser');
+const {
+    INDEX_ROUTE,
+    LOGIN_ROUTE,
+    LOGOUT_ROUTE,
+    EXPECTED_USERNAME,
+    EXPECTED_PASSWORD,
+    FIRST_PARTY_STATIC_FILE_SUFFIX,
+    THIRD_PARTY_STATIC_FILE_SUFFIX
+} = require("../../utils/constants");
+
+class OriginApp {
```

Comment:
Not necessary here since you already have `servers/nodejs/app.js` which is the actual server